### PR TITLE
Change column layout in order overview. Closes #302

### LINKED ISSUE #302: Order overview: Improve columns
The labels of the columns are unclear and should be improved, maybe with explaining tooltips. Also, a reordering of the columns could help.

### PR DESCRIPTION
**Pull request by rixx**
2016-11-26-132808_1167x608_scrot

### CODE REVIEW

**Comment by raphaelm** [comment]
Yay! Now that this looks so much nicer, what do you think about expired orders? I don't even remember correctly if they are in the "pending" column and added to the total or not, which is a pretty bad sign. :( Should we add them as their own column now?

**Comment by rixx** [comment]
*sigh* We probably should.

**Comment by rixx** [comment]
Done in second commit.

File: src/pretix/control/templates/pretixcontrol/orders/overview.html [hunk 1]
```diff
@@ -17,63 +17,78 @@
...
+                                    {{ item.num_pending|togglesum }}
                                 </a>
                             </td>
                             <td>
                                 <a href="{{ listurl }}?item={{ item.id }}&amp;status=p&amp;provider={{ item.provider }}">
                                     {{ item.num_paid|togglesum }}
                                 </a>
                             </td>
+                            <td>
+                                <a href="{{ listurl }}?item={{ item.id }}&amp;provider={{ item.provider }}">
```
Comment (raphaelm) [inline src/pretix/control/templates/pretixcontrol/orders/overview.html]: Shouldn't this now link to something like ``status=np`` that we would first need to create?

Comment (rixx) [inline src/pretix/control/templates/pretixcontrol/orders/overview.html]: Hm, true - or we just don't add a link there. Seems sensible to me, too.

**Comment by raphaelm** [comment]
Thanks! :)

**Pull request merged**